app/scripts/modules/core/authentication/userMenu/userRolesModal.controller.ts

```diff
@@ -0,0 +1,18 @@
+import {module} from 'angular';
+import {IModalServiceInstance} from 'angular-ui-bootstrap';
+
+class UserRolesModalCtrl {
+
+  static get $inject() { return ['$uibModalInstance', 'userRoles']; }
+
+  constructor ( private $uibModalInstance: IModalServiceInstance,
```

Comment:
Also, it doesn't look like you're using this in the modal's template. Please remove if it's not being used.